tests/foreman/cli/test_leapp_client.py

```diff
@@ -252,109 +59,27 @@ def test_leapp_upgrade_rhel(
-    'upgrade_path',
-    [
-        {'source_version': RHEL8_VER, 'target_version': RHEL9_VER},
-    ],
-    ids=lambda upgrade_path: f'{upgrade_path["source_version"]}'
-    f'_to_{upgrade_path["target_version"]}',
-)
-def test_leapp_upgrade_rhel_non_admin(
-    module_target_sat,
-    module_sca_manifest_org,
-    default_location,
-    custom_leapp_host,
-    upgrade_path,
-    verify_target_repo_on_satellite,
-    precondition_check_upgrade_and_install_leapp_tool,
-):
-    """Test to upgrade RHEL host to next major RHEL release using leapp preupgrade and leapp upgrade
-    job templates
-
-    :id: afd295ca-4b0e-439f-b880-ae92c300fd9f
-
-    :BZ: 2257302
-
-    :customerscenario: true
-
-    :steps:
-        1. Import a subscription manifest and enable, sync source & target repositories
-        2. Create LCE, Create CV, add repositories to it, publish and promote CV, Create AK, etc.
-        3. Register content host with AK
-        4. Verify that target rhel repositories are enabled on Satellite
-        5. Update all packages, install leapp tool and fix inhibitors
-        6. Create a non-admin user with "Organization admin", "Remote Execution Manager" and "Remote Execution User" role assigned to it.
-        7. Run Leapp Preupgrade and Leapp Upgrade job template from the user created in step 6.
-
-    :expectedresults:
-        1. Update RHEL OS major version to another major version from non-admin user role.
-    """
-    login = gen_string('alpha')
-    password = gen_string('alpha')
-    roles = ['Organization admin', 'Remote Execution Manager', 'Remote Execution User']
+    login = settings.server.admin_username
+    password = settings.server.admin_password
     org = module_sca_manifest_org
-    user = module_target_sat.cli_factory.user(
-        {
-            'admin': False,
-            'login': login,
-            'password': password,
-            'organization-ids': org.id,
-            'location-ids': default_location.id,
-        }
-    )
-    for role in roles:
-        module_target_sat.cli.User.add_role({'id': user['id'], 'login': login, 'role': role})
-
+    if 'auth_type' == 'non-admin':
+        login = gen_string('alpha')
+        password = gen_string('alpha')
+        roles = ['Organization admin', 'Remote Execution Manager', 'Remote Execution User']
+        user = module_target_sat.cli_factory.user(
+            {
+                'admin': False,
+                'login': login,
+                'password': password,
+                'organization-ids': org.id,
+                'location-ids': default_location.id,
+            }
+        )
+        for role in roles:
+            module_target_sat.cli.User.add_role({'id': user['id'], 'login': login, 'role': role})
+    # Workaround for https://issues.redhat.com/browse/RHEL-55871
+    assert custom_leapp_host.execute('echo \'ulimit -n 16384\' > /root/.bashrc').status == 0
+    assert custom_leapp_host.execute('source /root/.bashrc').status == 0
```

Comment:
```suggestion
    ```suggestion
    if is_open('RHEL-55871'):
         assert custom_leapp_host.execute('echo \'ulimit -n 16384\' > /root/.bashrc').status == 0
```
```

Reply:
I was going to implement the same but we can use is_open for `SAT-12345` issue only.

Reply:
why?

Reply:
You can find this here: https://github.com/SatelliteQE/robottelo/blob/8b0e3e033502140f70f258911d21e6076a1b8122/robottelo/utils/issue_handlers/__init__.py#L7

Reply:
@Gauravtalreja1 Shweta is right, I'm working on it, will raise pr soon

Reply:
@jameerpathan111 Great, could you handle this in your PR then, so we could unblock the merge of this PR?

Reply:
Sure. I have draft pr https://github.com/SatelliteQE/robottelo/pull/16348 raised but it'll take some time to test and finish it. We can merge this one.

Reply:
Thanks Jameer. I am resolving this conversation now.